examples/src/main/java/com/mapbox/navigation/examples/core/FeedbackButtonActivity.kt

```diff
@@ -298,7 +300,8 @@ class FeedbackButtonActivity : AppCompatActivity(), OnMapReadyCallback,
                 feedback.description,
                 UI,
                 screenShot,
-                feedback.feedbackSubType.toTypedArray()
+                feedback.feedbackSubType.toTypedArray(),
+                AppMetadata.Builder(BuildConfig.APPLICATION_ID, BuildConfig.VERSION_NAME).build()
```

Comment:
make the build configs default values in the builder.

default
``` kotlin
AppMetadata.Builder().build()
```
custom
``` kotlin
AppMetadata.Builder()
  .applicationId(BuildConfig.APPLICATION_ID)
  .build()
```

Reply:
There are no reasonable default values that we could use for `name` and `version` fields.

Reply:
ok, i was thinking BuildConfig.APPLICATION_ID, BuildConfig.VERSION_NAME were reasonable. 

but if not, also happy with them being required.